narwhal/benchmark/benchmark/commands.py

```diff
@@ -33,11 +33,16 @@ def generate_key(filename):
         assert isinstance(filename, str)
         return f'./narwhal-node generate_keys --filename {filename}'
 
+    @staticmethod
+    def get_pk(filename):
```

Comment:
nit: changing this to `get_pub_key` might help clear up any confusion between pk meaning private or public